src/views/Dissolutions/ReviewConfirmDissolution.vue

```diff
@@ -114,7 +114,7 @@
                 <p id="effective-date-time-instructions" class="info-text">
                   Select the date and time of the dissolution of the Company. You may select a date
                   up to 10 days in the future (note: there is an <strong>additional fee of
-                  {{futureEffectiveFeePrice}}</strong> to enter a dissolution date in the future).
+                  $100.00</strong> to enter a dissolution date in the future).
```

Comment:
It's always $100.00 🤷

Reply:
![Screen Shot 2021-12-13 at 8 37 54 AM](https://user-images.githubusercontent.com/53542131/145852020-7af650ac-033b-4971-8359-b2a1ff7687d4.png)

Reply:
I believe this was designed to fetched from the Pay API and then displayed here. For some reason, `futureEffectiveFeePrice` was null... possibly the fees were not fetched when the app started up (they did at one time).

My first choice would be to try to restore the functional code. If that's not possible then I'm OK with the hard-coded value.

Reply:
Ah, good call. 
Fixed the bug in `app.vue` to populate dynamically. 
Even though this value is constant, it could be useful to have the feePrices information locally.